engine/baml-lib/baml/tests/validation_files/class/secure_types.baml

```diff
@@ -17,207 +17,45 @@ class ComplexTypes {
   o (((int | string) | bool[]), (float, double) | long_long_identifier_123)
 }
 
-// error: Type `apple_pie` does not exist. Did you mean one of these: `ComplexTypes`, `float`, `bool`, `string`, `int`?
+// error: Error validating: This line is not a valid field or attribute definition. A valid class property looks like: 'myProperty string[] @description("This is a description")'
```

Comment:
this error got much worse.

seems we broke the grammar :(

<blocking>

Reply:
you reviewed an outdated version of the PR - the grammar wasn't broken, the test was still using the `{key: value}` map syntax